packages/react/src/portal.tsx

```diff
@@ -123,6 +129,19 @@ function ChildrenToFBO({
       store.setState({ size: { width, height, top: 0, left: 0 } })
     })
   })
+
+  useEffect(() => {
+    const unsubscribe = store.subscribe((state) => {
```

Comment:
3 small things:
1. you can directly return the result of the store.subscribe
2. its better to only update the projection matrix if aspect or camera changed (e.g. `(state, prevState) => state.camera != prevState.camera ....`)
3. your code only works with a PerspectiveCamera, use the same code as R3F to make it also work with a OrthographicCamera